fe/fe-core/src/main/java/org/apache/doris/nereids/rules/analysis/BindSink.java

```diff
@@ -140,13 +147,28 @@ public List<Rule> buildRules() {
                                                     column.getName()
                                             ));
                                         } else {
-                                            columnToOutput.put(column.getName(),
-                                                    new Alias(Literal.of(column.getDefaultValue())
-                                                            .checkedCastTo(DataType.fromCatalogType(column.getType())),
-                                                            column.getName()));
+                                            if (column.getDefaultValueExpr() == null) {
+                                                columnToOutput.put(column.getName(),
+                                                        new Alias(Literal.of(column.getDefaultValue())
+                                                                .checkedCastTo(
+                                                                        DataType.fromCatalogType(column.getType())),
+                                                                column.getName()));
+                                            } else {
+                                                Expression defualtValueExpression = FunctionBinder.INSTANCE.rewrite(
+                                                        new NereidsParser().parseExpression(
+                                                                column.getDefaultValueExpr().toSql()),
+                                                        new ExpressionRewriteContext(ctx.cascadesContext));
+                                                NamedExpression slot = defualtValueExpression instanceof NamedExpression
+                                                        ? ((NamedExpression) defualtValueExpression)
+                                                        : new Alias(defualtValueExpression);
+
+                                                columnToOutput.put(column.getName(), slot);
+                                            }
                                         }
                                     }
                                 }
+                            } catch (Exception e) {
+                                throw new AnalysisException(e.getMessage(), e.getCause());
```

Comment:
do not add catch here, only try catch in the smallest scope

Reply:
yes, I change the try-catch scope.